### DISCUSSION TITLE
No signal in output

### DISCUSSION THREAD

**Discussion by ilyamordasov** [fhdm-dev/scoppy#101] [Scoppy Front end designs]
- [ ] Ordered a pcb from oshwlab FScope 250k5-V2 and soldered according to BOM list and schematic. When I connected generated signal to ch1 I got nothing in scoppy app. Also if I disconnect wires the signal is not changed, but if I select 1.25 MHz square signal I see the noisy line with increased frequency, but it doesn't look like square signal, even f I select sine signal. Where could be a problem? I use scoppy-pico-fscope-250k5-v14.uf2 firmware

**Comment by fhdm-dev** [comment]
Does the oscilloscope work if you apply a DC voltage to the input directly? If not then you'll need to check all the voltages from the input to the Pico ADC to find out where the problem is.

**Reply by ilyamordasov** [reply]
I checked my soldering and fixed where it was needed. So, after that if I connect sig1 (direct) to pin 26, I can see my expected signal. But if I connect sig1 to ch1 (H1) I see nothing

UPD: I've started to check all path from bnc/H1 so, guess i know how to catch an issue

UPD2: I can't see the signal after c11 and r1, before them I see

**Reply by fhdm-dev** [reply]
That's the inverting input of the op-amp so I would expect you would always see 0V there (because the non-inverting input is connected to GND).

Do you see your signal at the output of the first op-amp? If you apply 1V DC at the BNC connector you should see -0.33V at pin1 of U1. Also, check the supply voltages at pins 4 and 8. They should be -2.3 (approximately) and 3.3V.

**Reply by ilyamordasov** [reply]
Unfortunately I don't see negative voltage. Pin 1 and 4 give me 0.49v, bnc 1.65. Also I checked u5, where I expect to get -2.5v, but I get 0.15v

**Reply by ilyamordasov** [reply]
Also I checked `vsys` pin, it has 0.25v, guess it is too low. When I tied `vsys` to `3.3V`, I got my signal in output

**Reply by fhdm-dev** [reply]
Yes - that's way too low. I would expect VSYS to be about 5V. If connected to a phone the actual voltage will depend on the charge of the battery and could range between 4.5V and 5.5V.

**Reply by ilyamordasov** [reply]
I checked with phone and tablet, in both cases `vsys` is equal zero. Maybe because the dev-kit is clone from AliExpress. So, I think I will use it with connected to 3.3V. Is it possible to measure the signal, let say, 10V, for example, using your kit?

**Reply by fhdm-dev** [reply]
I'm not sure the board will work correctly with 3.3V on VSYS. You might see some clipping of the signal....Sorry but I don't what dev-kit you are referring to.
You can use the FSCOPE board with 10X probes to measure voltages higher than 6V. You can change the probe settings in the app by tapping the channel badge at the bottom of the screen, then tap Settings and then Probe.
